test/feature/test_lightglue_onnx.py

```diff
@@ -11,19 +11,24 @@
 
 
 @pytest.mark.skipif(ort is None, reason="OnnxLightGlue requires onnxruntime-gpu")
-@pytest.mark.skipif(not torch.cuda.is_available(), reason="OnnxLightGlue requires CUDA")
 class TestOnnxLightGlue:
     @pytest.mark.slow
-    @pytest.mark.parametrize("weights", OnnxLightGlue.MODEL_URLS.keys())
-    def test_pretrained_weights(self, weights):
-        model = OnnxLightGlue(weights)
+    @pytest.mark.skipif(not torch.cuda.is_available(), reason="Test requires CUDA")
+    @pytest.mark.parametrize("weights", ["disk", "superpoint", "disk_fp16", "superpoint_fp16"])
+    def test_pretrained_weights_cuda(self, weights):
+        model = OnnxLightGlue(weights, "cuda")
         assert model is not None
 
     @pytest.mark.slow
-    def test_forward(self):
-        model = OnnxLightGlue()
+    @pytest.mark.parametrize("weights", ["disk", "superpoint"])
+    def test_pretrained_weights_cpu(self, weights):
+        model = OnnxLightGlue(weights, "cpu")
+        assert model is not None
+
+    @pytest.mark.slow
+    def test_forward(self, device):
```

Comment:
no dtype ?

Reply:
The input dtype used is only `float32`, for both FP32 and FP16 models.